src/Uno.UWP/System/VirtualKeyHelper.iOS.cs

```diff
@@ -0,0 +1,135 @@
+﻿using System;
+using Foundation;
+using UIKit;
+
+namespace Windows.System
+{
+	internal static class VirtualKeyHelper
+	{
+		public static VirtualKey FromKeyCode(UIKeyboardHidUsage keyCode)
+			=> keyCode switch
```

Comment:
You had fun to do that??? 😅

Reply:
LOL yeah lot of fun finding the matching keys